source/tools/testConsistency/ClientApp/startProject.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
+cd ../nodejs-renderer
+START flask run 
+cd ../dotnetWPF-renderer
+START flask run -p 4000
```

Comment:
>-p 4000 [](start = 16, length = 7)

What does this port (?) signify? Also we should have some comments here explaining what this script does and what it's used for. Comments here but also a readme.md to explain how to use/run this

Reply:
Added comments in the bat file and in the readme